cmip6_downscaling/methods/common/utils.py

```diff
@@ -10,7 +11,11 @@
 from xarray_schema import DataArraySchema
 from xarray_schema.base import SchemaError
 
-from cmip6_downscaling.methods.common.containers import BBox
+import cmip6_downscaling.methods.common.containers as containers
```

Comment:
I'm curious why this was needed?

Reply:
There was some circular import problem where utils needed an import from containers and containers needed an import from utils.

Reply:
huh! okay, I think you can leave these as is then. Perhaps we should pull the `str_to_hash` function further up in the call stack to avoid this.